### ISSUE TITLE
Testing Sign in With Apple results in `cannot unmarshal string into Go struct field UserInfo.email_verified of type bool` when requesting email scope

### ISSUE THREAD

**Issue by PatrickMilroy** [auth0/auth0-cli#840]
### Checklist

- [X] I have looked into the [README](https://github.com/auth0/auth0-cli#readme) and have not found a suitable solution or answer.
- [X] I have looked into the [documentation](https://auth0.github.io/auth0-cli/) and have not found a suitable solution or answer.
- [X] I have searched the [issues](https://github.com/auth0/auth0-cli/issues) and have not found a suitable solution or answer.
- [X] I have upgraded to the [latest version](https://github.com/auth0/auth0-cli/releases/latest) of this project and the issue still persists.
- [X] I have searched the [Auth0 Community](https://community.auth0.com/c/sdks/5) forums and have not found a suitable solution or answer.
- [X] I agree to the terms within the [Auth0 Code of Conduct](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md).

### Description

When testing client and using an `Apple` social provider and requesting the scopes `openid,profile,email`, The browser login succeeds, but in the CLI I receive an error

```
▸    failed to fetch user info: cannot decode response: json: cannot unmarshal string into Go struct field UserInfo.email_verified of type bool
```

If I do not request the scope `email` the process suceeds and returns userinfo, an access token and id token. 

I have tested sucessfuly tested the same client using both a Google Connection and Auth0 Username/Password Database connection while requesting the scopes `openid,profile,email` 

### Expectation

To be returned an access token, an id token and userinfo relating to the `email` scope (eg email, email_verified)

### Reproduction

1. Given an Application/Client and a Apple Social Connection
2. When running `auth0 test login --scopes openid,profile,email` and using the Apple Social Connection 
3. The browser login succeeds, but the CLI returns the error above

### Auth0 CLI version

1.0.1

**Comment by willvedd** [comment]
@PatrickMilroy Thanks for opening this ticket. Apple an identify provider returns a string encoded boolean value for `email_verified` as opposed to other IdPs which return a normal boolean. I'm going to work with the Go SDK folks to get this resolved.

**Comment by willvedd** [comment]
This has been fixed with [v1.1.1](https://github.com/auth0/auth0-cli/releases/tag/v1.1.1). Please upgrade and give it a go!

**Comment by PatrickMilroy** [comment]
@willvedd Thanks! Testing and working
